plugin/parser/schema_diff_test.go

```diff
@@ -12,33 +11,61 @@ import (
 )
 
 func TestComputeDiff(t *testing.T) {
-	oldSchema, err := parser.Parse(parser.Postgres, parser.ParseContext{}, `
-CREATE TABLE projects ();
-`)
-	require.NoError(t, err)
-	newSchema, err := parser.Parse(parser.Postgres, parser.ParseContext{}, `
-CREATE TABLE users (
+	tests := []struct {
+		engineType parser.EngineType
+		oldSchema  string
+		newSchema  string
+		want       string
+		err        error
+	}{
+		{
+			engineType: parser.Postgres,
+			oldSchema:  `CREATE TABLE projects ();`,
+			newSchema: `CREATE TABLE users (
 	id serial PRIMARY KEY,
 	username TEXT NOT NULL
 );
 CREATE TABLE projects ();
 CREATE TABLE repositories (
 	id serial PRIMARY KEY
-);
-`)
-	require.NoError(t, err)
-
-	got, err := parser.SchemaDiff(oldSchema, newSchema)
-	require.NoError(t, err)
-
-	// The DDLs for the diff should be in the exact same order as the DDLs in the new schema.
-	want := `CREATE TABLE users (
+);`,
+			want: `CREATE TABLE users (
 	id serial PRIMARY KEY,
 	username TEXT NOT NULL
 );
 CREATE TABLE repositories (
 	id serial PRIMARY KEY
 );
-`
-	assert.Equal(t, want, got)
+`,
+			err: nil,
+		},
+		{
+			engineType: parser.Postgres,
+			oldSchema:  `CREATE TABLE projects ();`,
+			newSchema: `CREATE TABLE projects (
+	id serial PRIMARY KEY
+);`,
+			// FIXME(@joe): this is an unwanted result.
```

Comment:
This is currently expected result because we don't support column diff yet

Reply:
Removed, PTAL https://github.com/bytebase/bytebase/pull/2706